framework/console/Controller.php

```diff
@@ -296,6 +307,18 @@ public function options($actionID)
     }
 
     /**
+     * Returns option alias names.
+     * Child classes may override this method to specify alias options.
+     *
+     * @return array the options alias names valid for the action
```

Comment:
for example : "where the keys is option name and value is alias name for option"?